packages/editor/src/components/document-tools/index.js

```diff
@@ -68,11 +70,15 @@ function DocumentTools( {
 			hasFixedToolbar: getSettings().hasFixedToolbar,
 			showIconLabels: get( 'core', 'showIconLabels' ),
 			isDistractionFree: get( 'core', 'distractionFree' ),
+			isVisualMode: getEditorMode() === 'visual',
+			isZoomedOutView: __unstableGetEditorMode() === 'zoom-out',
```

Comment:
Well, these selectors will be confusing when they're stabilized 😄

Reply:
Indeed 😬